pages/integrations/docker_hub.md.erb

```diff
@@ -102,7 +102,7 @@ images in use you may continue to hit Docker Hub rate limits.
 
 ## Running a read-through caching registry
 
-There are two popular options for running a private caching docker registry,
+There are two popular options for running a private caching registry,
```

Comment:
I mean, this is specifically about docker though?

Reply:
I'm glad you asked about this, because I goofed on this. I made this change to avoid allowing "Docker registry" (as in a generic registry for Docker images), barring the enforcement of "Docker Registry" (as in a _specific_ Docker Inc.-built registry implementation). But upon closer review, it turns out that _all_ the uses of "Docker Registry" in context ought to be "Docker registry". I've undone the changes for "Docker Registry" with d40ad341a439fe038f13176a829dde722522267b